instrumentation/lettuce/lettuce-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v4_0/LettuceSingletons.java

```diff
@@ -38,14 +39,14 @@
             .addAttributesExtractor(attributesExtractor)
             .newInstrumenter(SpanKindExtractor.alwaysClient());
 
-    LettuceConnectNetAttributesExtractor connectNetAttributesExtractor =
-        new LettuceConnectNetAttributesExtractor();
+    NetClientAttributesExtractor<RedisURI, Void> netClientAttributesExtractor =
+        NetClientAttributesExtractor.create(new LettuceConnectNetAttributesGetter());
```

Comment:
Can that `LettuceConnectNetAttributesGetter` instance be reused? The extractor itself can probably be just inlined.

Reply:
Yup! Good call.